tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java

```diff
@@ -116,7 +116,13 @@
             "unitPreferenceData");
     private static final PathMatcher CURRENCY_DATA_PATHS =
         supplementalMatcher("currencyData");
-    private static final PathMatcher NUMBERING_SYSTEMS_PATHS =
+    private static final Predicate<CldrPath> UNITS_DATA_PATHS =
+        supplementalMatcher(
+            "convertUnits",
+            "unitConstants",
+            "unitQuantities",
+            "unitPreferenceData");
+    private static final Predicate<CldrPath> NUMBERING_SYSTEMS_PATHS =
```

Comment:
Nope. This is painful / won't work - since we're stale w.r.t. upstream/master. I'm removing this file from this PR.